README.md

```diff
@@ -48,32 +48,30 @@ Afterwards run it with:
 
 ## Database
 
-Database consists of three sections:
-
-- `categories` - which informs on what categories data of points is divided
-- `visible_data` - list of categories which will be visible by application users
-- `data` - actual data split into `categories`
-
-
-### `categories`
-Fully configurable map where key is name of category and value is list of allowed types. E.g.
-* "car_elements": ["mirror", "wheel", "steering wheel"]
-* "color": ["red", "blue", "green"]
-
-### `data`
-Data consists of two parts:
-* obligatory and constant
-  * `name` - name of the object
-  * `position` - coordinates of object
-* category dependent - depending on your `categories` setup it varies. See example of config below.
-
-#### `custom data`
-You can define your own, more complex data types as dictionary.
-* obligatory fields in dictionary:
-  * `type` - type of data
-  * `value` - value of data
-* optional fields in dictionary:
-  * `displayValue` - value to display instead of `value`
+The database is stored in JSON, in the `map` section. For an example database see `tests/e2e_tests/e2e_test_data.json`. The first subsection `data` consists of the actual datapoints, representing points on a map.
+
+Datapoints have fields. The next subsections define special types of fields:
+- `obligatory_fields` - here are explicitely stated all the fields that the application assumes are presnt in all datapoints. E.g.
```

Comment:
Now this `obligatory_fields` is not used at all in code right? If so - add TODO comment to point it out.
Also - what about adding it to `e2e_test_data.json`?

Reply:
Yes it is not used in the code for now. Should I add this TODO comment in this readme?
Sure, I can add this subsection to example `e2e_test_data.json`.

Reply:
Yes. Add it in Readme, and please add it to example.